spec/codecs/json_spec.rb

```diff
@@ -64,6 +64,48 @@
         insist { decoded } == true
       end
     end
+
+    context "when json could not be parsed" do
+
+      let(:message)    { "random_message" }
+
+      it "add the failure tag" do
+        subject.decode(LogStash::Json.dump(message)) do |event|
```

Comment:
no need to do `LogStash::Json.dump(message)`. This is done internally in the codec